source/Releases/Release-Foxy-Fitzroy.rst

```diff
@@ -288,6 +288,36 @@ There are copies in `example_interfaces <https://index.ros.org/p/example_interfa
 - ``std_msgs/msg/UInt8``
 - ``std_msgs/msg/UInt8MultiArray``
 
+sros2
+^^^^^
+
+Use of security enclaves
+""""""""""""""""""""""""
+
+As of Foxy, domain participants are not mapped directly to ROS nodes anymore.
+The concept of security enclave was introduced, an enclave is a process or group of processes that will share the same identity and access control rules.
+Each process using ROS Security is part of an enclave.
```

Comment:
It's not entirely clear what you mean by this statement, can you clarify?

Reply:
I believe this one can be removed if we keep the next one.
My goal was to say that processes using security will always load an enclave's security material and that the enclave name need to be specified (as opposed to infer from node name and namespace

Reply:
I'm fine on the next one, should we remove this one then?